### PR TITLE
Chore/release202004

### PR DESCRIPTION
**Pull request by atharvar28**

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# covid19.datacommons.io/manifest.json
## :warning: Services on branch
- portal

File: covid19.datacommons.io/etlMapping.yaml [hunk 1]
```diff
@@ -12,6 +12,8 @@ mappings:
       - name: latitude
       - name: longitude
       - name: FIPS
+      - name: iso2
```
Comment (themarcelor) [inline covid19.datacommons.io/etlMapping.yaml:15]: That doesn't seem to belong to this pr.

**Comment by PlanXCyborg** [comment]
# covid19.datacommons.io/manifest.json
## :warning: Services on branch
- portal

**Comment by atharvar28** [comment]
Need to divide into two PRS
